covidapi/import_data.py

```diff
@@ -13,21 +22,17 @@ def main():
     Base.metadata.create_all(engine)
     db_instance = SessionLocal()
 
-    with open(CSV_FILE) as csv_file:
-        csv_reader = csv.reader(csv_file, delimiter=',')
-        next(csv_reader, None)  # skip the headers
-
-        for row in csv_reader:
-            dr = DailyReport(
-                province_state=row[0],
-                country_region=row[1],
-                last_update=datetime.fromisoformat(row[2]),
-                confirmed=row[3],
-                deaths=row[4],
-                recovered=row[5],
-            )
-
-            db_instance.add(dr)
+    for row in get_data_with_caching('02-29-2020'):
+        dr = DailyReport(
+            province_state=row['Province/State'],
+            country_region=row['Country/Region'],
+            last_update=datetime.fromisoformat(row['Last Update']),
```

Comment:
How do we specify the date format? ie: now we seem to have `02-29-2020` --> `month-day-year`, is it a fix format or do we have to specify it?

Reply:
The `month-day-year` format is used in the filenames, and also some of the very old reports use this for `last_update`, but the script doesn't handle these yet.

At some point they switched to the iso format, so that's what we're still using to parse `last_update`.

Annoyingly there is 1 report since the 29th Feb which inexplicably reverted to the old format. This PR was getting quite complicated so I decided to skip that date until it's fixed or we can make the code work with both formats.